server.js

```diff
@@ -2,22 +2,137 @@ const express = require("express");
 const cors = require("cors");
 
 const app = express();
+app.use(express.json())
 
 app.use(cors());
 
-const welcomeMessage = {
-  id: 0,
-  from: "Bart",
-  text: "Welcome to CYF chat system!",
-};
+const messages = [
+  {
+    id: 0,
+    from: "Bart",
+    text: "Welcome to CYF chat system!",
+    
+  }]
 
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+// const messages = [welcomeMessage];
 
 app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
+//////////////////////Create a new message//////////////////////////// 
+ app.post('/messages/', (req, res) =>{
+  let newId = messages[messages.length -1].id + 1
```

Comment:
Taking the ID based on the length of the array can be err-prone very early if you are not careful. What if you have 2 messages and delete the first one then create a new one. Could it be then duplicated? What other side effects could cause for instance when deleting a message by ID?